ODLA/platforms/dnnl/odla_dnnl.cc

```diff
@@ -655,82 +655,46 @@ odla_value odla_Mul(odla_value lhs, odla_value rhs, const odla_value_id id) {
   return binary_eltwise(dnnl::algorithm::binary_mul, lhs, rhs, id);
 }
 
-odla_value odla_Round(odla_value input, const odla_value_id id) {
-  auto ret_md = input->mem.get_desc();
-  auto ret_mem = dnnl::memory(ret_md, g_comp->eng);
-  auto desc = dnnl::eltwise_forward::desc(dnnl::prop_kind::forward_inference,
-                                          dnnl::algorithm::eltwise_round,
-                                          input->mem.get_desc());
-  auto pd = dnnl::eltwise_forward::primitive_desc(desc, g_comp->eng);
-  auto prim = dnnl::eltwise_forward(pd);
-
-  add_op(prim, {{DNNL_ARG_SRC, input->mem}, {DNNL_ARG_DST, ret_mem}});
-
+static odla_value eltwise_op(dnnl::algorithm algo, odla_value input,
```

Comment:
how about rename it to something like "unary_eltwise_op" ? so ppl won't confuse it with "binary_eltwise_op"